bindings/pulp/bindings/server_info.py

```diff
@@ -51,12 +52,3 @@ def get_distributors(self):
         """
         path = self.base_path + 'distributors/'
         return self.server.GET(path)
-
-    def ping(self):
-        """
-        Retrieves basic status information from the server.
-
-        @return: Response
-        """
-        path = '/v2/services/status/'
-        return self.server.GET(path)
```

Comment:
@beav you are correct this should just be removed. The webcode handler is there, but it hasn't been wired up in application.py in a long time. There is no way for a web request to reach this code.